src/classes/frSubscription.cls

```diff
@@ -116,9 +189,11 @@ public class frSubscription extends frModel implements frSyncable {
             String salesforceFieldName = funraiseFieldName.replaceAll(regex, replacement);
             salesforceFieldName =+ 'funraise__' + salesforceFieldName.capitalize() + '__c';
             Schema.SObjectField field = fields.get(salesforceFieldName);
+            if(field == NULL) continue; //added null handling
```

Comment:
Can we discuss why this was added?  In general I think the field being null would be pretty exceptional since it should be saved based on us introspecting the fields availalbe in frSetupController.  If we're going to add handling on this, can an exception be logged in a Error__c object